### PR TITLE
[Feature] Add doris engine to linkis

### PR DESCRIPTION
**Pull request by ChengJie1053**
[Feature] Add doris engine to linkis

Support cvs, json, parquet, orc format file import doris

Submit sample
{
    "executionContent": {"code": "{\"linkis.ec.doris.stream.load.file.path\":\"/Users/chengjie/Downloads/doris/test.csv\",\"linkis.ec.doris.database\":\"demo\",\"linkis.ec.doris.table\":\"example_tbl\"}", "runType":  "doris"},
    "params": {
                    "variable": {},
                    "configuration": {
                            "runtime": {
                                "linkis.ec.doris.host":"192.168.201.54",
                                "linkis.ec.doris.2pc.enabled":true,                           
                                "linkis.ec.doris.http.port":8030
                                }
                            }
                    },
    "source":  {"scriptPath": "file:///mnt/bdp/hadoop/1.sql"},
    "labels": {
        "engineType": "doris-1.2.6",
        "userCreator": "linkis-IDE"
    }
}

Execution result

image

image

image

image

image

### CODE REVIEW

**Comment by peacewong** [comment]
Thank you, can you use json as code to pass? Parameters that do not apply to runtime Map

File: linkis-computation-governance/linkis-manager/linkis-label-common/src/main/java/org/apache/linkis/manager/label/conf/LabelCommonConfig.java [hunk 1]
```diff
@@ -75,6 +75,9 @@ public class LabelCommonConfig {
   public static final CommonVars<String> NEBULA_ENGINE_VERSION =
       CommonVars.apply("wds.linkis.nebula.engine.version", "3.0.0");
 
+  public static final CommonVars<String> DORIS_ENGINE_VERSION =
+      CommonVars.apply("wds.linkis.doris.engine.version", "1.2.6");
```
Comment (peacewong) [inline linkis-computation-governance/linkis-manager/linkis-label-common/src/main/java/org/apache/linkis/manager/label/conf/LabelCommonConfig.java:79]: remove wds prefix

File: linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/conf/DorisConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+package org.apache.linkis.engineplugin.doris.conf;
+
+import org.apache.linkis.common.conf.CommonVars;
+
+public class DorisConfiguration {
+
+  public static final CommonVars<Integer> ENGINE_CONCURRENT_LIMIT =
+      CommonVars.apply("linkis.engineconn.concurrent.limit", 100);
+
+  public static final CommonVars<Integer> ENGINE_DEFAULT_LIMIT =
```
Comment (peacewong) [inline linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/conf/DorisConfiguration.java:27]: should  'linkis.engineconn.doris'  prefix?

File: linkis-engineconn-plugins/doris/src/main/resources/log4j2.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+  
+<configuration status="error" monitorInterval="30">
+    <appenders>
+        <Console name="Console" target="SYSTEM_OUT">
```
Comment (peacewong) [inline linkis-engineconn-plugins/doris/src/main/resources/log4j2.xml:21]: should use RollingFile

File: linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/conf/DorisConfiguration.java (hunk 2 above)
Comment (ChengJie1053) [reply linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/conf/DorisConfiguration.java:27]: Ok, thanks for reviewing the code

File: linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/executor/DorisEngineConnExecutor.java [hunk 4]
```diff
@@ -0,0 +1,653 @@
...
+    if (StringUtils.isBlank(code)) {
+      throw new DorisException(
+          DorisErrorCodeSummary.DORIS_CODE_IS_NOT_BLANK.getErrorCode(),
+          DorisErrorCodeSummary.DORIS_CODE_IS_NOT_BLANK.getErrorDesc());
+    } else {
+      realCode = code.trim();
+    }
+    logger.info("Doris engine begins to run code:\n {}", realCode);
+
+    checkRequiredParameter(code);
```
Comment (peacewong) [inline linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/executor/DorisEngineConnExecutor.java:187]: should use realCode?

File: linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/conf/DorisConfiguration.java (hunk 2 above)
Comment (peacewong) [reply linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/conf/DorisConfiguration.java:27]: this need optimize,thanks

File: linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/executor/DorisEngineConnExecutor.java (hunk 4 above)
Comment (ChengJie1053) [reply linkis-engineconn-plugins/doris/src/main/java/org/apache/linkis/engineplugin/doris/executor/DorisEngineConnExecutor.java:187]: Ok, thanks for reviewing the code

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**